versions.json

```diff
@@ -248,17 +238,17 @@
     "version": "v0.19.2",
     "title": "v0.19",
     "aliases": [
-      "v0.19.0",
-      "v0.19.1"
+      "v0.19.1",
+      "v0.19.0"
     ]
   },
   {
     "version": "v0.18.3",
     "title": "v0.18",
     "aliases": [
       "v0.18.2",
-      "v0.18.1",
-      "v0.18.0"
+      "v0.18.0",
+      "v0.18.1"
```

Comment:
nit: I'm just curious why the order of these versions is changed. Is it needed for some reason?

Reply:
I think it's a formatter thing

Reply:
I noticed it too, not sure why.. I just ran the `mike` command. I actually run it from the same container image that is used in the gh workflow so it should be exactly the same environment. wild guess is that the delete command (which I used now, but is rarely used otherwise) does a more extensive manipulation of the file than the deploy command (that we use regularly when building the docs.